man/ostree.repo-config.xml

```diff
@@ -143,6 +143,18 @@ Boston, MA 02111-1307, USA.
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>lock-timeout</varname></term>
+        <listitem>
+          <para>
+            Integer value controlling the default repository locking
+            timeout in seconds. If the value is <literal>-1</literal>,
```

Comment:
What if the value is 0 or <-1? Could go with “If the value is negative then ostree will block…”

Reply:
0 means don't wait - true non-blocking. I thought that was implied, but I could spell it out. < -1 is an error. I could just do all negative values, but I liked having the explicit value. It also allowed me to later use -2 as a sentinel.